kolibri/core/content/test/test_public_api.py

```diff
@@ -95,6 +95,25 @@ def test_schema_version_too_high(self):
         )
         self.assertEqual(response.status_code, 400)
 
+    def test_import_metadata_none_pk(self):
+        response = self.client.get(
+            reverse(
+                "kolibri:core:importmetadata-detail",
+                kwargs={"pk": None},
+            )
+        )
+        self.assertEqual(response.status_code, 404)
```

Comment:
This would indeed return a 400 as it would be caught first by try catch block. moving forward with this is the same behaviour in Studio lets change this to 400.